### ISSUE TITLE
More poll options

### ISSUE THREAD

**Issue by detiDevs** [Wadzek1/WadzekBot#1]
It should be possible to create a poll with more than two options to answer.
You could write:
```
!poll <numberOfOptions>
```

I think this would be a nice feature

EDIT: And it would be cool if there was a feature to abstain. The WadzekBot could then add an "X"-Emoji or something like that. This could be turned on or off by using a parameter, like:

```
!poll <numberOfOptions> <skipVotePossible>
```

For that parameter you could then write "skip" or "noSkip" or simply 1 or 0.

**Comment by kiriDevs** [comment]
@Wadzek1 Create this feature on a new branch, a "feature-branch", that you then pull-request into main. Be sure to link this issue to the pull request before merging it. That way, it will show which pull-request implemented this and therefore closed the issue.

**Comment by kiriDevs** [comment]
@detiDevs Please remember to label future issues accordingly. I did it for you this time.

**Comment by detiDevs** [comment]
> @detiDevs Please remember to label future issues accordingly. I did it for you this time.

Okay

**Comment by kiriDevs** [comment]
> For that parameter  you could then write "skip" or "noskip" or simply 1 or 0

You'd use `true` and `false`, but I get the idea.

@Wadzek1 When you link a pull request, don't merge it yet - we can add that feature in the same feature-branch.

**Comment by kiriDevs** [comment]
@Wadzek1 Would you mind if I added the `true` and `false` options for abstaining votes?

**Comment by kiriDevs** [comment]
@detiDevs Please add additional requests in own issues instead of editing one you already had. We were about to mere code and close this issue when we saw the edit.
If you reference another issue, as in this case (because you are explaining the command with things only implemented after this issue), you can link it by typing `#[issueID]` - here: `#1`, which will turn into this link: #1

**Comment by kiriDevs** [comment]
@detiDevs PR is now up; if you want to contribute something yourself, let us know and give us a PR into `feature/poll_more-options`. If you don't, that's okay as well. Just wanted to let you know.

**Comment by kiriDevs** [comment]
@detiDevs Was merged to main in #2, which also included #3. 
Issue was closed with merging of this pull request.
